tests/test_multiple_interpreters.py

```diff
@@ -19,9 +19,9 @@ def test_independent_subinterpreters():
     if sys.version_info >= (3, 15):
         import interpreters
```

Comment:
When accepting the PEP the steering council made this `concurrent.interpreters`, and it's been given an exception to land in 3.14.0b3. Not sure there's any action worth taking yet, once b3 is out we can just drop support for b2, I assume. Unless we want to go all in on importorskip. Normally not a fan of importorskip for version dependent things, but maybe we should go that way instead.

Reply:
b3 just got tagged, so I'm going to see if I can test this locally on my build.